plugins/modules/remote_management/lenovoxcc/xcc_redfish_command.py

```diff
@@ -0,0 +1,599 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: xcc_redfish_command
+short_description: Manages Lenovo Out-Of-Band controllers using Redfish APIs
+version_added: 2.2.0
```

Comment:
2.2.0 has already been released, so nothing can be added to it anymore:
```suggestion
version_added: 2.3.0
```